src/Vue.jl

```diff
@@ -50,7 +50,10 @@ function vue(template, data=Dict(); kwargs...)
             onjs(v, @js function (val)
                 # This copy is needed to avoid the Vue.js reactivity system
                 @var valcopy = JSON.parse(JSON.stringify(val))
-                this.vue[$skey] = valcopy
+                if val != this.vue[$skey]
```

Comment:
See the checks in the WebIO code you linked to. This will always return true if val is NaN, and if, val and `this.vue[$skey]` are both arrays with the same contents, e.g. `["a", "b"]`